pyquil/tests/test_device.py

```diff
@@ -58,11 +132,11 @@ def test_isa():
             Edge(targets=[2, 0], type="CPHASE", dead=None),
         ])
     d = isa.to_dict()
-    assert d == ARCH_QPU
+    assert d == isa_dict
 
 
-def test_kraus_model():
-    km = KrausModel('I', (5.,), (0, 1), [np.array([[1 + 1j]])], 1.0)
+def test_kraus_model(kraus_model_I):
```

Comment:
can we make this test more like `test_isa`? that one really seems to cover all the bases, and is built from the dictionary fixture